packages/config/config/devices/0x0258/nas-ab01z.json

```diff
@@ -82,6 +82,10 @@
 				{
 					"label": "5 minutes",
 					"value": 3
+				},
+				{
+					"label": "Always on",
+					"value": 255
```

Comment:
please update the `maxValue` to match or (even better) remove min/maxValue and add `"allowManualEntry": false` before the options.